relayer-cli/src/commands/query/channel.rs

```diff
@@ -121,7 +121,7 @@ mod tests {
     #[test]
     fn parse_channel_query_end_parameters() {
         let default_params = QueryChannelEndCmd {
-            chain_id: Some("ibc0".to_string().parse().unwrap()),
```

Comment:
Why is this needed?

Reply:
I didn't implement this as part of my issue, maybe something merged from master? Maybe @ancazamfir knows ?